spec.yaml

```diff
@@ -17,6 +17,25 @@ info:
 
     Feedback on the API as well as this documentation is welcome via `support@giantswarm.io` or on IRC channel [#giantswarm](irc://irc.freenode.org:6667/#giantswarm) on freenode.
 
+    ## Idempotency
+
+    Idempotency is the concept of making API calls repeatable, while guaranteeing that the state changes associated with the API call are only executed once.
+    Idempotency allows you to simply repeat certain API requests in case of an error.
+
+    Currently the only operation supporting idempotency is the [creation of clusters](#operation/addCluster), but other operations will likely follow in the future.
```

Comment:
I checked the cluster-service and saw that we already do this for the deletion as well. See https://github.com/giantswarm/cluster-service/blob/ce61d2ef05081fc70f1810519f68fc9e0397447b/service/cluster/deleter/service.go#L116.